core-bundle/tests/Security/User/UserCheckerTest.php

```diff
@@ -107,9 +107,9 @@ public function testThrowsAnExceptionIfTheAccountIsNotActiveYet(): void
         $user = $this->mockClassWithProperties(FrontendUser::class);
         $user->username = 'foo';
         $user->locked = 0;
-        $user->disable = '';
-        $user->login = '1';
-        $user->start = $time;
+        $user->disable = false;
+        $user->login = true;
+        $user->start = (string) $time;
```

Comment:
`$user->start` is a `varchar` in the database - and it will stay a string when loaded from the database. As this test simulates a model returned from the database, the value should be a string, not an integer.